Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,17 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return tasks
+    .map(taskToDurations => taskToDurations.duration / 60)
```

Comment:
The names you chose for this parameter is a bit strange. I would expect `taskToDurations` to be the name of a function or method instead of a parameter. I would call this parameter just `task` because it is one item from the `tasks` array.

Reply:
Dear Wilgert , thank you for the feedback. I must be more careful when I select the names. I made the changes according to your feedback.